pkg/alertmanager/amcfg.go

```diff
@@ -1122,6 +1122,7 @@ func (cb *configBuilder) convertPushoverConfig(ctx context.Context, in monitorin
 		Message:       in.Message,
 		URL:           in.URL,
 		URLTitle:      in.URLTitle,
+		TTL:           string(*in.TTL),
 		Priority:      in.Priority,
 		HTML:          in.HTML,
 	}
```

Comment:
you need to check for nil
```suggestion
		Priority:      in.Priority,
		HTML:          in.HTML,
	}
	
	if in.TTL != nil {
		out.TLL = *in.TTL
	}
```

Reply:
aha I missed that